openedx/features/job_board/tests/views/test_job_board.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.sites.models import Site
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from openedx.core.djangoapps.theming.models import SiteTheme
+from openedx.features.job_board.views import JobCreateView, JobListView
+from openedx.features.job_board.tests.factories import JobFactory
+
+
+class JobBoardViewTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(JobBoardViewTest, cls).setUpClass()
+        site = Site(domain='testserver', name='test')
+        site.save()
+        theme = SiteTheme(site=site, theme_dir_name='philu')
+        theme.save()
+
+    def test_job_create_view(self):
+        response = self.client.get(reverse('job_create'))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(JobCreateView.fields, '__all__')
+
+    def test_job_detail_view_invalid_pk(self):
+        response = self.client.get(reverse('job_detail', kwargs={'pk': 1}))
+        self.assertEqual(response.status_code, status.HTTP_302_FOUND)
+
+    def test_job_detail_view_valid_pk(self):
+        job = JobFactory()
+        job.save()
```

Comment:
Please verify `JobFactory()` create and save object, is `save()` redundant?

Reply:
Good catch